tests/scripts/remote_monitoring_tests/rest_apis/test_update_recommendations.py

```diff
@@ -0,0 +1,273 @@
+import pytest
+from helpers.fixtures import *
+from helpers.kruize import *
+from helpers.list_reco_json_validate import *
+from helpers.utils import *
+
+
+@pytest.mark.sanity
+def test_update_valid_recommendations_after_results_after_create_exp(cluster_type):
+    input_json_file = "../json_files/create_exp.json"
```

Comment:
Include description for all the tests included with expected behavior

Reply:
done